src/kind.ts

```diff
@@ -0,0 +1,74 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
+
+export class KindJob extends Job {
+    constructor(name: string, image?: string) {
+        if (image == undefined) {
+            image = kindJobImage;
+        }
+        super(name, image);
+
+        // kind needs to run as a privileged pod
+        this.privileged = true;
+
+        // kind needs to add the following volumeMounts to function properly
+        // the Brigade project must enable `allowMountHosts`
+        this.volumes = [
```

Comment:
I'm super curious as to what each volume mount enables -- could we add a note to each?